.github/workflows/run-integration-tests.yml

```diff
@@ -0,0 +1,18 @@
+name: Run integration tests
+
+on:
+  pull_request:
```

Comment:
We should run this on commits to `develop` and `master` as well

Reply:
Should we also run them on release branches?

Reply:
Release branches should already have PRs associated with them, so the other trigger should work fine there. `master` and `develop` are the only two long-running branches that we have without PRs